src/Symfony/Component/Security/Core/Resources/translations/security.hr.xlf

```diff
@@ -76,7 +76,7 @@
             </trans-unit>
             <trans-unit id="20">
                 <source>Too many failed login attempts, please try again in %minutes% minutes.</source>
-                <target state="needs-review-translation">Previše neuspjelih pokušaja prijave, pokušajte ponovo za %minutes% minutu.|Previše neuspjelih pokušaja prijave, pokušajte ponovo za %minutes% minuta.</target>
+                <target>Previše neuspjelih pokušaja prijave, pokušajte ponovo za %minutes% minutu.|Previše neuspjelih pokušaja prijave, pokušajte ponovo za %minutes% minute.|Previše neuspjelih pokušaja prijave, pokušajte ponovo za %minutes% minuta.</target>
```

Comment:
Shouldn't translation 19 be changed for plural as well?

Reply:
Possibly yes, since according to the code, 20 is used only for values greater than 1, which means zero uses 19 as well. However, zero doesn't really make sense as a value in this context.

https://github.com/symfony/symfony/blob/095d2864f0a63754e3408d99ee80385dc3347b2c/src/Symfony/Component/Security/Core/Exception/TooManyLoginAttemptsAuthenticationException.php#L45

Reply:
The translations for this item are split in 3 items for legacy reasons, to preserve compat with existing translations :shrug: